### DISCUSSION TITLE
Auto Dark Mode stops working after changing time zone

### DISCUSSION THREAD

**Discussion by AdionC** [AutoDarkMode/Windows-Auto-Night-Mode#704] [Q&A] [answered]
Noticed this a few times now, and I think Auto Dark Mode stops working after the time zone changed, and then doesn't work anymore until the computer is restarted.

**Comment by Spiritreader** [comment, marked as answer]
C# applications are by default unable to recognize time zone changes until the application is restarted.  The current TimeZone is cached to prevent erroneous behavior for time-critical applications.

If you are taking your computer across time zones while it is in standby or hibernate without having it fully shut down beforehand, you're running into a currently unsupported scenario.

You may open a feature request (enhancement) if you would like to see that functionality included.
